aten/src/ATen/native/native_functions.yaml

```diff
@@ -1421,7 +1421,7 @@
     CPU: _embedding_bag_forward_only_cpu
     CUDA: _embedding_bag_forward_only_cuda
 
-- func: rowwise_prune(Tensor weight, Tensor mask, ScalarType compressed_indices_dtype) -> (Tensor, Tensor)
+- func: _rowwise_prune(Tensor weight, Tensor mask, ScalarType compressed_indices_dtype) -> (Tensor, Tensor)
```

Comment:
I think it also makes sense to put this in the nn module, you can do that via "python_module: nn" -- note you'll have to access it via ._nn._C or something.

Reply:
The plan is to silence this change as we don't want to expose this to PyTorch frontend as this operator doesn't have detailed documentation/API yet. Do you still think we need to add "python_module:nn" in this case?

Reply:
@gchanan : If we are planning to access this for now only in C++, do we need to expose it via nn?